cypress/e2e/hub/approvals.cy.ts

```diff
@@ -1,12 +1,46 @@
 import { Approvals } from './constants';
+import { randomString } from '../../../framework/utils/random-string';
 
 describe('Approvals', () => {
+  let thisCollectionName: string;
+  let namespace: string;
+
   before(() => {
     cy.hubLogin();
+    cy.getNamespace('ibm');
+    cy.navigateTo('hub', Approvals.url);
+  });
+
+  after(() => {
+    cy.deleteCollection(thisCollectionName, 'ibm', 'published');
   });
 
   it('it should render the approvals page', () => {
+    cy.verifyPageTitle(Approvals.title);
+  });
+
+  it('user can upload a new collection and approve it', () => {
+    thisCollectionName = 'test' + randomString(4).toLowerCase();
+    namespace = 'ibm';
+
+    cy.uploadCollection(thisCollectionName, namespace);
     cy.navigateTo('hub', Approvals.url);
     cy.verifyPageTitle(Approvals.title);
+    cy.clickTableRowPinnedAction(thisCollectionName, 'approve');
```

Comment:
This command is not working in my local tests, there may be some problem with syncing. My tip is that it gets the old row, but then filter is applied and old row is removed, so it cant click it...